packages/enterprise/src/license.ts

```diff
@@ -4,18 +4,26 @@ import path from "path";
 import fetch from "node-fetch";
 import type Stripe from "stripe";
 import pino from "pino";
-import { omit, pick, sortBy } from "lodash";
+import { pick, sortBy } from "lodash";
 import AsyncLock from "async-lock";
 import { stringToBoolean } from "shared/util";
 import { ProxyAgent } from "proxy-agent";
-import { LicenseDocument, LicenseModel } from "./models/licenseModel";
+import { LicenseModel } from "./models/licenseModel";
 
-export const LICENSE_SERVER =
-  "https://central_license_server.growthbook.io/api/v1/";
+export const LICENSE_SERVER = "http://localhost:8080/api/v1/";
+//"https://central_license_server.growthbook.io/api/v1/";
```

Comment:
I have a test that protects against this. I originally didn't want to make it an env var, because then someone could easily change their env vars to a fake license server without violating the enterprise package's license.  But now we do the signature check always, so they can't do it.  So I've done as you suggested and allowed the LICENSE_SERVER_URL to be set from an env var, and falls back to the actual license server.